scripts/centos_jswarm.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+CINCH="$(readlink -f "$(dirname "$0")/../")"
+CENTOS_VERSION="${1}"
+INVENTORY="${CINCH}/inventory/cent${CENTOS_VERSION}_jswarm_docker"
+if [ ! -e "${INVENTORY}" ]; then
+	echo "You must specify a CentOS version to continue"
+	exit 1
+fi
+# Extracts the version line from the setup.py script, and trims off the rest of the line to leave
+# only the expected version
```

Comment:
Please add an example of what the expected version string should look like in this comment.